### PR TITLE
Add 1604 cross arm64

### PR DESCRIPTION
**Pull request by jashook**
Add 16.04 rootfs build under `18.04/cross/arm64/16.04`

Addresses @janvorli comment. This will complete the 18.04 docker images that get created, we can now build 18.04 -> 18.04 arm64 and 18.04 -> 16.04 with this change.

### CODE REVIEW

File: src/ubuntu/16.04/crossdeps/Dockerfile [hunk 1]
```diff
@@ -34,13 +34,3 @@ RUN apt-get update \
         llvm-9 \
         python-lldb-6.0 \
     && rm -rf /var/lib/apt/lists/*
-
-# Install x86_arm crossgen dependencies
-RUN dpkg --add-architecture i386 \
```
Comment (janvorli) [inline src/ubuntu/16.04/crossdeps/Dockerfile]: Are you sure we don't need it anymore? I have though that we use x86 crossgen to crossgen stuff for ARM.

Comment (jashook) [reply src/ubuntu/16.04/crossdeps/Dockerfile]: Pretty sure, we should generate arm from an x64 build @echesakovMSFT to confirm.

Comment (echesakovMSFT) [reply src/ubuntu/16.04/crossdeps/Dockerfile]: Yes, on Linux we are running x64-arm crossgen, so we don't need this dependency.

**Comment by jashook** [comment]
I cannot seem to find where the add-architecture i386 is being added. @MichaelSimons the failures exist in HEAD

**Comment by MichaelSimons** [comment]
@mthalman - can you please help @jashook with the CI failures?

**Comment by mthalman** [comment]
The build issues were just flaky build stuff with ARM machines.  It looks like the Ubuntu 18.04 pipeline is getting really close to the default timeout of 1 hr.  So that should be increased.

@jashook - Can you add a `linuxAmdBuildJobTimeout: 90` parameter to [dotnet-buildtools-prereqs-ubuntu18.yml](https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/master/eng/pipelines/dotnet-buildtools-prereqs-ubuntu18.yml)?  You can look at [dotnet-buildtools-prereqs-centos7.yml](https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/master/eng/pipelines/dotnet-buildtools-prereqs-centos7.yml) as an example of where such a parameter is set.

**Comment by jashook** [comment]
I will keep the change removing the x86 rootfs, it is currently unused and bloats the build images. If we ever need it back I suggest adding a new image which contains it.

**Comment by jashook** [comment]
Done thank you @mthalman

**Comment by mthalman** [comment]
@jashook - Is this ready for merge or are you still waiting for feedback?

**Comment by jashook** [comment]
It is worth getting a few more people to buy off on the change @jkoritzinsky @safern @ViktorHofer ptal

**Comment by jashook** [comment]
@mthalman I believe this is ready to be merged

**Comment by jashook** [comment]
@MichaelSimons were there images produced after merge?

**Comment by MichaelSimons** [comment]
> were there images produced after merge?

deferring to @mthalman

**Comment by mthalman** [comment]
> @MichaelSimons Michael Simons (.NET) FTE were there images produced after merge?

Yes, you can find the tags for the built images, by looking at the [image info file](https://github.com/dotnet/versions/blob/master/build-info/docker/image-info.dotnet-dotnet-buildtools-prereqs-docker-master.json).  In this case, two commits were made:
* https://github.com/dotnet/versions/commit/1e42b09de6adc40dc0d723426268f014acd13317#diff-d9067485273c757fce700f0561c6bc12
* https://github.com/dotnet/versions/commit/c7d8e9717795f466b74f0e21a1bb73590ed84a2b#diff-d9067485273c757fce700f0561c6bc12

**Comment by jashook** [comment]
These are missing `src/ubuntu/18.04/cross/arm64/16.04` and `src/ubuntu/18.04/cross/arm/16.04` that were added to the manifest.

**Comment by jashook** [comment]
@mthalman could you help provide insight on what went wrong? Thank you!

**Comment by mthalman** [comment]
Sorry about that.  Those commits were for a different PR.  Looks like the builds failed due to infra problems.  I'll nudge them along and reply when images are ready.

**Comment by mthalman** [comment]
Ok, here are the commits with the new tags:
* https://github.com/dotnet/versions/commit/722a203ca9f1a38d831ea7cfaa0de5a58c03a47a#diff-d9067485273c757fce700f0561c6bc12
* https://github.com/dotnet/versions/commit/106e4a55d7f85a1c4b671d3fe09817a47b282960

**Comment by jashook** [comment]
Perfect thank you!

**Pull request merged**